nomos-services/blend/src/backends/libp2p.rs

```diff
@@ -116,7 +116,8 @@ impl BlendBehaviour {
             blend:
                 nomos_blend_network::Behaviour::<SphinxMessage, TokioIntervalStreamProvider>::new(
                     nomos_blend_network::Config {
-                        duplicate_cache_lifespan: 60,
+                        // This size must be ROUNDS_IN_SESSION * BLEND_HOPS
+                        seen_message_cache_size: 1_944_000,
```

Comment:
I think it would be better for this to be a constant that links to the spec paragraph that defines it.